# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1251790299 -7200
#      Tue Sep 01 09:31:39 2009 +0200
# Branch release-3-2-x
# Node ID 295001bff26d55cd987fb3d07c314be2a235a692
# Parent  48a9c78cfbc94c37ced29ffd667ec06a50754bcb
datestr: add missing semicolon

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-08-29  John W. Eaton  <jwe@octave.org>
+
+	* time/datestr.m: Add missing semicolon.
+
 2009-08-26  E. Joshua Rigler  <relgire@gmail.com>
 
 	* time/datestr.m: Set tm.isdst to -1 before calling mktime.
 
 2009-08-06  John W. Eaton  <jwe@octave.org>
 
 	* statistics/base/std.m: Correctly work along singleton dimension.
 	From Christoph Ellenberger <C.Ellenberger@gmx.net>.
diff --git a/scripts/time/datestr.m b/scripts/time/datestr.m
--- a/scripts/time/datestr.m
+++ b/scripts/time/datestr.m
@@ -273,17 +273,17 @@ function retval = datestr (date, f, p)
     tm.mon = vi(2) - 1;
     tm.mday = vi(3);
     tm.hour = vi(4);
     tm.min = vi(5);
     sec = vi(6);
     tm.sec = fix (sec);
     tm.usec = fix (rem (sec, 1) * 1e6);
     ## Force mktime to check for DST.
-    tm.isdst = -1
+    tm.isdst = -1;
  
     str = strftime (df, localtime (mktime (tm)));
 
     if (i == 1)
       retval = str;
     else
       retval = [retval; str];
     endif
